wagtail_bucketav/urls.py

```diff
@@ -4,4 +4,6 @@
 
 app_name = "wagtail_bucketav"
 
-urlpatterns = [path("sns-hook/", BucketAVWebhookView.as_view(), name="sns-hook")]
+urlpatterns = [
+    path("sns-hook/", BucketAVWebhookView.as_view(), name="sns-hook"),
```

Comment:
more readable when each URL pattern lives on its own line?  
(but that's very subjective, happy to revert it of course :slightly_smiling_face: )